src/Oscoin/Crypto.hs

```diff
@@ -0,0 +1,8 @@
+module Oscoin.Crypto
+    ( RealWorldCrypto
+    , MockCrypto(..)
+    ) where
+
+data RealWorldCrypto
+
+data MockCrypto = MockCrypto
```

Comment:
Question: is there a reason this is here, instead of under the `Test.*` hierarchy?

Reply:
The reason being we need to import it inside the `Oscoin.Crypto.Hash.Mock` and `Oscoin.Crypto.PubKey.Mock`. Unless we also move them into the test hierarchy, we need this guy here.

Reply:
I see I see. Let's leave it then for now :+1: